tests/core/test_config_substitutor.py

```diff
@@ -0,0 +1,261 @@
+from contextlib import contextmanager
+from unittest import mock
+
+import pytest
+
+from great_expectations.core.config_substitutor import (
+    ConfigurationSubstitutor,
+    secretmanager,
+)
+
+
+@pytest.fixture
+def config_substitutor() -> ConfigurationSubstitutor:
+    return ConfigurationSubstitutor()
+
+
+@contextmanager
+def does_not_raise():
+    yield
+
+
+@pytest.mark.parametrize(
+    "input_value,method_to_patch,return_value",
+    [
+        ("any_value", None, "any_value"),
+        ("secret|any_value", None, "secret|any_value"),
+        (
+            "secret|arn:aws:secretsmanager:region-name-1:123456789012:secret:my-secret",
+            "great_expectations.core.config_substitutor.ConfigurationSubstitutor.substitute_value_from_aws_secrets_manager",
+            "success",
+        ),
+        (
+            "secret|projects/project_id/secrets/my_secret",
+            "great_expectations.core.config_substitutor.ConfigurationSubstitutor.substitute_value_from_gcp_secret_manager",
+            "success",
+        ),
+        (
+            "secret|https://my-vault-name.vault.azure.net/secrets/my_secret",
+            "great_expectations.core.config_substitutor.ConfigurationSubstitutor.substitute_value_from_azure_keyvault",
+            "success",
+        ),
+    ],
+)
+def test_substitute_value_from_secret_store(
```

Comment:
It looks like these tests were just moved over from `tests/data_context/test_data_context_utils.py`?
If it is not arduous can you mark these tests as `unit`, `integration`, or `e2e` as appropriate?

Reply:
done!